src/components/HomePage/SearchArea/AdvancedSearch/FieldOptions.js

```diff
@@ -1,9 +1,9 @@
 /* istanbul ignore file */
 
-export default Object.freeze([
-    { label: "Back-End", value: "BACK_END" },
-    { label: "Front-End", value: "FRONT_END" },
-    { label: "Dev-Ops", value: "DEVOPS" },
-    { label: "Machine Learning", value: "ML" },
-    { label: "Computer Vision", value: "COMPUTER_VISION" },
-]);
+export default Object.freeze({
+    "BACKEND": "Back-End",
+    "FRONTEND": "Front-End",
+    "DEVOPS": "Dev-Ops",
+    "ML": "Machine Learning",
+    "COMPUTER_VISION": "Computer Vision",
```

Comment:
There is no COMPUTER_VISION in the backend aswell, only an "OTHER" field, so I am not sure if this is intended. To check this, the file from the backend is: "FieldTypes.js"